### PR TITLE
Added mapping documentation into es exporter readme

### PR DESCRIPTION
**Pull request by michalpristas**
**Description:** 
Adding mapping documentation to the doc

**Link to tracking Issue:** -

**Testing:**  -

**Documentation:** exporter/elasticsearchexporter/README.md

Thinking about automating this but having compound mapping i'm not finding any common way of specifying these. 
Will leave it for a folloup

### CODE REVIEW

**Review by JaredTan95 — APPROVED** [review]

File: exporter/elasticsearchexporter/README.md
Comment (atoulme) [inline exporter/elasticsearchexporter/README.md:218]: Should this be linking to semantic conventions?

File: exporter/elasticsearchexporter/README.md
Comment (atoulme) [inline exporter/elasticsearchexporter/README.md:267]: Should that not be `otlp/unknown` ?

File: exporter/elasticsearchexporter/README.md
Comment (atoulme) [inline exporter/elasticsearchexporter/README.md:258]: Can those component values contain the character `/` ?

File: exporter/elasticsearchexporter/README.md
Comment (atoulme) [inline exporter/elasticsearchexporter/README.md:275]: Is there a precedence?

File: exporter/elasticsearchexporter/README.md
Comment (atoulme) [reply exporter/elasticsearchexporter/README.md:218]: should this conversion be in the semantic-conventions specification?

File: exporter/elasticsearchexporter/README.md
Comment (atoulme) [inline exporter/elasticsearchexporter/README.md:224]: Do you mean that for `host.name`, we will duplicate the attribute to `host.hostname`?

Is this treatment only true for the `host.name` attribute? Can this be handled as a separate paragraph?

File: exporter/elasticsearchexporter/README.md
Comment (michalpristas) [reply exporter/elasticsearchexporter/README.md:218]: i don't think so

File: exporter/elasticsearchexporter/README.md
Comment (michalpristas) [reply exporter/elasticsearchexporter/README.md:258]: i dont see why not, but having `telemetry.sdk.language/name/version` none of them has this so far

File: exporter/elasticsearchexporter/README.md
Comment (michalpristas) [reply exporter/elasticsearchexporter/README.md:275]: yes in the order i mentioned them. do you think we could make it more clear/explicit?

File: exporter/elasticsearchexporter/README.md
Comment (michalpristas) [reply exporter/elasticsearchexporter/README.md:224]: so far it is just for host.hostname.
i went with this format just in case we have more exceptions later

File: exporter/elasticsearchexporter/README.md
Comment (michalpristas) [reply exporter/elasticsearchexporter/README.md]: made it explicit

**Comment by michalpristas** [comment]
@atoulme does it look good to you?

File: exporter/elasticsearchexporter/README.md
Comment (atoulme) [reply exporter/elasticsearchexporter/README.md:224]: the column header says "Preserve" but you use the word "Preserved". If this is exceptional treatment, maybe it needs to be called out more. I also don't understand exactly what you write. Do you mean this?
```suggestion
When "Preserved" is true, the attribute will be preserved in the payload and duplicated as mapped to its ECS equivalent.
```

File: exporter/elasticsearchexporter/README.md
Comment (atoulme) [reply exporter/elasticsearchexporter/README.md:258]: It'd be great that it doesn't happen in the future too.

File: exporter/elasticsearchexporter/README.md
Comment (atoulme) [reply exporter/elasticsearchexporter/README.md:218]: I think it should. Here is the equivalent for log mapping for Splunk HEC: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model-appendix.md#splunk-hec

Comment (michalpristas) [reply exporter/elasticsearchexporter/README.md:218]: i see, there's also an ECS specific mapping also defined there: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model-appendix.md#elastic-common-schema

do you suggest adding a link here pointing to that mapping

Comment (michalpristas) [reply exporter/elasticsearchexporter/README.md:222]: added a backlink to this readme

**Comment by michalpristas** [comment]
@atoulme @bogdandrutu any steps required to proceed?

**Review by atoulme — APPROVED** [review]
Looks OK, a codeowner of the exporter should approve as well

**Review by carsonip — APPROVED** [review]

**Review by ycombinator — APPROVED** [review]
Thanks for adding this conversion/mapping documentation, @michalpristas!  

cc: @carsonip @lahsivjar for awareness, for when more such conversions are added to the ES exporter.

**Pull request merged**
